<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWDialog</class>
 <widget class="QDialog" name="QWDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QWDialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>64</y>
     <width>299</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>299</width>
     <height>81</height>
    </size>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QCheckBox" name="chkBoxBold">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>30</y>
      <width>49</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Bold</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkBoxItalic">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>30</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Italic</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkBoxUnder">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>30</y>
      <width>73</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Underlin</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>151</y>
     <width>299</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>299</width>
     <height>51</height>
    </size>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="rBtnBlack">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>30</y>
      <width>87</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Black</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rBtnBlue">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>30</y>
      <width>87</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rBtnRed">
    <property name="geometry">
     <rect>
      <x>199</x>
      <y>30</y>
      <width>87</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Red</string>
    </property>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>208</y>
     <width>299</width>
     <height>59</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>297</width>
     <height>59</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>274</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancle">
   <property name="geometry">
    <rect>
     <x>226</x>
     <y>274</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>332</x>
     <y>274</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>chkBoxBold</tabstop>
  <tabstop>chkBoxItalic</tabstop>
  <tabstop>chkBoxUnder</tabstop>
  <tabstop>rBtnBlack</tabstop>
  <tabstop>rBtnBlue</tabstop>
  <tabstop>rBtnRed</tabstop>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>btnOK</tabstop>
  <tabstop>btnCancle</tabstop>
  <tabstop>btnExit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>QWDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancle</sender>
   <signal>clicked()</signal>
   <receiver>QWDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnExit</sender>
   <signal>clicked()</signal>
   <receiver>QWDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
